firebase/functions/test/test-utils.ts

```diff
@@ -13,7 +13,7 @@ function sleep(ms: number) {
 export const attempt = async function(operation: () => Promise<any>): Promise<any> {
     let attempts: number = 0;
     let error: Error;
-    while (attempts < 5) {
```

Comment:
Are the firestore calls unreliable? Just curious why we need so many attempts for them.

Reply:
The tests sometimes check a condition that is satisfied by some asynchronous event, e.g. something triggers a Cloud Function in the background, and we are interested in the result of the function. The name could probably be improved here, but this is essentially attemptFutureCondition.